addons/sale/views/product_attribute_views.xml

```diff
@@ -58,7 +61,8 @@
         <field name="inherit_id" ref="product.product_template_attribute_value_view_form"/>
         <field name="arch" type="xml">
             <field name="name" position="after">
-                <field name="html_color" readonly="True" />
+                <field name="display_type" invisible="1"/>
+                <field name="html_color" readonly="True"  attrs="{'invisible': [('display_type','in', ['radio', 'select'])]}"/>
```

Comment:
Same as above. Also, 128 chars is starting to be quite a long line (does not fit on the screen on GitHub), consider splitting it on the attributes ;)